apps/labs/next.config.js

```diff
@@ -30,18 +30,56 @@ const nextConfig = {
       },
     ];
   },
-  async rewrites() {
+  async redirects() {
     return [
       {
-        source: '/plausible/js/script.js',
-        destination: 'https://plausible.io/js/script.js',
-      },
-      {
-        source: '/plausible/api/event',
-        destination: 'https://plausible.io/api/event',
+        source: '/about',
+        destination: '/',
+        permanent: true,
       },
     ];
   },
+  async rewrites() {
+    // The conventions of this return value are described in the Next.js docs:
+    // https://nextjs.org/docs/api-reference/next.config.js/rewrites
+    return {
+      afterFiles: [
+        // These rewrites are checked after pages/public files are checked but
+        // before dynamic routes.
+
+        // Proxying Plausible through Vercel
+        // https://plausible.io/docs/proxy/guides/vercel
+        {
+          source: '/plausible/js/script.js',
+          destination: 'https://plausible.io/js/script.js',
+        },
+        {
+          source: '/plausible/api/event',
+          destination: 'https://plausible.io/api/event',
+        },
+      ],
+      fallback: [
+        // These rewrites are checked after both pages/public files and dynamic
+        // routes are checked.
+        {
+          source: '/:file(.+\\..+)',
+          destination: 'https://quansight-labs.netlify.app/:file',
+        },
+        {
+          // Why is /archive an anomaly on Netlify site?
+          // On Netlify, /2022 redirects to /2022/
+          // But then on same Netlify site, /archive/ redirects to /archive
+          // WTF.
+          source: '/archive',
+          destination: 'https://quansight-labs.netlify.app/archive',
+        },
+        {
+          source: '/:path*',
```

Comment:
Everything else falls back to the Netlify site. Note that this Vercel site does not use trailing slashes but the Netlify site does.